front/app/containers/HomePage/index.tsx

```diff
@@ -44,6 +52,18 @@ const HomePage = () => {
     }
   }, [pressedLetterAKey, userHasAdminAccess]);
 
+  if (
+    isHomepageBuilderEnabled &&
+    !isEmpty(homepageSettings?.data.attributes.craftjs_json)
+  ) {
+    return (
+      <>
+        <Viewer />
+        <Fragment name="pages/homepage_info/content" />
```

Comment:
What is the purpose of these `Fragment`?

Reply:
It's used to inject custom content. You can read more about it [here](https://www.notion.so/citizenlab/Fragments-8035a8a89f8b4b8186c1f73e624cc6a7).